conda-env/dev.yml

```diff
@@ -9,15 +9,16 @@ dependencies:
   - python=3.9.10
   - pip=22.0.3
   - beautifulsoup4=4.10.0
-  - cartopy=0.20.2
-  - cartopy_offlinedata=0.2.4
+  - cartopy=0.21.1
+  - cartopy_offlinedata=0.21.1
```

Comment:
Update the `cartopy` dependencies because the integration tests were failing due to https://github.com/SciTools/cartopy/issues/2086